runtime-decompiler/src/main/java/org/jrd/backend/data/Directories.java

```diff
@@ -7,6 +7,7 @@
     private static final String XDG_CONFIG_SUFFIX = File.separator + "conf";
     private static final String XDG_PLUGIN_SUFFIX = File.separator + "plugins";
     private static final String XDG_JRD_HOME = File.separator + ".config" + File.separator + "java-runtime-decompiler";
```

Comment:
here we go now, with duplicated code. Indeed it is!

Reply:
Imma take a wild guess at understanding and say that you want me to use XDG_JRD_SUFFIX in the initialization of XDG_JRD_HOME, yes?